selfdrive/car/hyundai/hyundaican.py

```diff
@@ -30,9 +31,20 @@ def create_lkas11(packer, apply_steer, steer_req, cnt, enabled, lkas11, hud_aler
   }
 
   dat = packer.make_can_msg("LKAS11", 0, values)[2]
-  dat = dat[:6] + dat[7]
-  checksum = hyundai_checksum(dat)
 
+  # CRC Checksum as seen on 2019 Hyundai Santa Fe
+  if checksum_type == 0:
```

Comment:
here you can use `CHECKSUM`, imported from values.py

Reply:
Please review now, I now pass fingerprint and imported values.py